iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties

```diff
@@ -27,21 +27,21 @@ cluster_name=defaultCluster
 ### Seed ConfigNode
 ####################
 
-cn_seed_config_node=127.0.0.1:10710
+cn_seed_config_node=nel-22:10710
```

Comment:
It also needs to be modified in iotdb-system.properties.template

Reply:
Reverted.